app/js/app.js

```diff
@@ -0,0 +1,172 @@
+// responsive canvas
+$(document).ready( function(){
+    //Get the canvas &
+    var c = $('#juice-canvas');
+    var ctx = c.get(0).getContext('2d');
+    var container = $(c).parent();
+
+    //Run function when browser resizes
+    $(window).resize( responsiveCanvas );
+
+    function responsiveCanvas(){
+        c.attr('width', $(container).width() ); //max width
```

Comment:
soft tabs set to two spaces throughout https://github.com/airbnb/javascript#whitespace--spaces